src/utils/cloudinary.ts

```diff
@@ -0,0 +1,10 @@
+import {v2 as cloudinary} from 'cloudinary'
+
+cloudinary.config({
+    cloud_name: "dddpzabl2",
+    api_key: "742787312133591",
+    api_secret: "-u2PGFMTHxtNJ4pyOl41y_mdnVw"
```

Comment:
Great job @gershomlapaix 
One suggestion I have for improving the code quality is to store sensitive information such as this API key and cloud name in environment variables, rather than hardcoding them in the codebase.